### PR TITLE
Push latest tag for alpine image and udpate documentation.

### PR DESCRIPTION
**Pull request by vijayendrabvs**
What this PR does
==============
This is a follow up patch for #659 - this pushes the lyft/envoy-alpine and lyft/envoy-alpine-debug images with the :latest tag. It also makes a note of these images in docs.

### CODE REVIEW

**Comment by vijayendrabvs** [comment]
@junr03 Have logged this as a follow up patch, thanks!

File: docs/install/sandboxes.rst [hunk 1]
```diff
@@ -315,3 +315,12 @@ the ``FROM`` line in any dockerfile.
 
 This will be particularly useful if you are interested in modifying envoy, and testing
 your changes.
+
+
+Using alpine envay image
```
Comment (junr03) [inline docs/install/sandboxes.rst]: typo

Comment (vijayendrabvs) [inline docs/install/sandboxes.rst]: Ah! Have fixed it, thanks!

File: docs/install/sandboxes.rst [hunk 2]
```diff
@@ -315,3 +315,12 @@ the ``FROM`` line in any dockerfile.
 
 This will be particularly useful if you are interested in modifying envoy, and testing
 your changes.
+
+
+Using alpine envoy image
+------------------------
```
Comment (junr03) [inline docs/install/sandboxes.rst]: can you actually add this info here: https://github.com/lyft/envoy/blob/11faea44b5b1c8ca8d868856190e65f2aaf64a9e/ci/README.md#developer-use-of-ci-docker-image

Comment (vijayendrabvs) [inline docs/install/sandboxes.rst]: @junr03 Sure, have moved it out of sandboxes.rst into ci/README.md.

**Comment by vijayendrabvs** [comment]
@junr03 is this good to go?

File: ci/README.md [hunk 3]
```diff
@@ -8,6 +8,13 @@ image at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image
 corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains
 an Envoy binary built from the latest tip of master that passed tests.
 
+# Using alpine envoy image
+
+A minimal image based on alpine is available as lyft/envoy-alpine.
```
Comment (junr03) [inline ci/README.md]: lyft/envoy-alpine -> `lyft/envoy-alpine`. Same below.

File: ci/README.md [hunk 4]
```diff
@@ -8,6 +8,13 @@ image at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image
 corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains
 an Envoy binary built from the latest tip of master that passed tests.
 
+# Using alpine envoy image
```
Comment (junr03) [inline ci/README.md]: Please mention as above that both this images have a `hash` tag and a `latest` tag.

Comment (vijayendrabvs) [inline ci/README.md]: @junr03 Thanks, I've changed the section organization to have two subsections for Ubuntu and Alpine images.

File: ci/README.md [hunk 5]
```diff
@@ -1,13 +1,33 @@
...
+may work with `lyft/envoy-build:latest` to provide a self-contained environment for building Envoy binaries and
+running tests that reflects the latest built Ubuntu Envoy image. Moreover, the Docker image
+at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image that has an Envoy binary at `/usr/local/bin/envoy`. The `<hash>`
+corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains an Envoy
+binary built from the latest tip of master that passed tests.
+
+## Alpine envoy image
+
+Minimal images based on alpine Linux allow for quicker deployment of Envoy. Two alpine based images are built,
+one with an Envoy binary with debug symbols and one stripped of them (and consequently the smallest of the built images).
```
Comment (junr03) [inline ci/README.md]: To make this section shorter I would just add to this paragraph the following: 

`...one with an Envoy binary with debug (`lyft/envoy-alpine-debug`) symbols and one stripped of them (`lyft/envoy-alpine`). Both images are pushed with two different tags: `<hash>` and `latest`. Parallel to the Ubuntu images above, `<hash>` corresponds to the master commit at which the binary was compiled, and `latest` corresponds to a binary built from the latest tip of master that passed tests.`

File: ci/README.md [hunk 6]
```diff
@@ -1,13 +1,33 @@
...
+
+The Alpine based Envoy Docker image at [`lyft/envoy-alpine:<hash>`](https://hub.docker.com/r/lyft/envoy-alpine/), and the
+Alpine based debug Envoy Docker image at [`lyft/envoy-alpine-debug:<hash>`](https://hub.docker.com/r/lyft/envoy-alpine-debug/) are
+both used for Travis CI checks, where `<hash>` is specified in [ci_steps.sh](https://github.com/lyft/envoy/blob/master/ci/ci_steps.sh).
```
Comment (junr03) [inline ci/README.md]: This is incorrect, the `<hash>` comes from the commit at which the image was built. But if you modify above as I suggest you can delete this paragraph.

File: ci/README.md [hunk 7]
```diff
@@ -1,13 +1,33 @@
...
+
+As with the Ubuntu based images, for the Alpine based images, developers
+may work with `lyft/envoy-alpine:latest` and `lyft/envoy-alpine-debug:latest` to provide a self-contained environment for
+building Envoy binaries and running tests that reflects the latest built Alpine Envoy image.
```
Comment (junr03) [inline ci/README.md]: This is incorrect, the images don't have an environment to build envoy, they just have an envoy binary in them. Again, if you take the edit from above you can just delete this paragraph.

File: docs/install/sandboxes.rst [hunk 8]
```diff
@@ -315,3 +315,4 @@ the ``FROM`` line in any dockerfile.
 
 This will be particularly useful if you are interested in modifying envoy, and testing
 your changes.
+
```
Comment (junr03) [inline docs/install/sandboxes.rst]: nit: extra newline

Comment (vijayendrabvs) [inline docs/install/sandboxes.rst]: @junr03 Thanks for catching the errors! I've made the changes as suggested.

**Comment by vijayendrabvs** [comment]
Thanks @junr03 !

**Pull request merged**